.github/actions/status-check/action.yml

```diff
@@ -0,0 +1,27 @@
+name: Check required jobs
+description: Check if all required jobs where successful or skipped
+
+inputs:
+  jobs:
+    description: Jobs (value must be toJSON(needs))
+    required: true
+
+runs:
+  using: "composite"
```

Comment:
```suggestion
  using: composite
```

Reply:
Copy/paste from [GH docs](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action), so leaving this one ;)

Reply:
Just a nit anyways :man_shrugging: